xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.ha.pmremote.integrate.guestsintocluster">
+   <title>Integrating KVM Guests into Cluster</title>
+   <para>To integrate the KVM guests into the cluster, do the following on
+    the physical host <systemitem class="domainname">&wsI;</systemitem>:</para>
+   <procedure>
+    <step>
+     <para>Start &pace;:</para>
+     <screen>&prompt.root;<command>systemctl</command> start pacemaker</screen>
+    </step>
+    <step>
+     <para>Check the status of the command <command>crm status</command>. It
+     should contain a running cluster with nodes that are all accessible.
+     </para>
+    </step>
+    <step>
+     <para>Check if all KVM guests nodes and IP addresses are listed
+     in the file <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>Dump the XML configuration of the KVM guest(s):
+     </para>
+     <screen>&prompt.root;<command>virsh</command> list --all
+ Id    Name         State
+-----------------------------------
+ -     &wsII;       shut off
+&prompt.root;<command>virsh</command> dump &wsII; > /etc/pacemaker/&wsII;.xml</screen>
+    </step>
+    <step>
+     <para>
+      Create a new <systemitem>VirtualDomain</systemitem> resource:
+      <remark>toms 2017-03-23: Check the command, this is probably not correct:</remark>
+     </para>
+     <screen>&prompt.root;<command>crm</command> configure
+&prompt.crm.conf;<command>primitive</command> vm.&wsII; ocf:heartbeat:VirtualDomain \
+  params hypervisor="qemu:///system" \
+         config="/etc/pacemaker/&wsII;.xml \
+  meta remote-node=&wsII;</screen>
+    </step>
+   </procedure>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.testing">
+  <title>Testing the Setup</title>
+  <para>To demonstrate how resources are executed, use a few dummy resources.
+   These dummy resources serve for testing purposes only.
+  </para>
+  <procedure>
+   <step>
+    <para>Create a few dummy resources:</para>
+    <screen>&prompt.root;<command>crm</command> configure primitive fake1 ocf:pacemaker:Dummy
+&prompt.root;<command>crm</command> configure primitive fake2 ocf:pacemaker:Dummy
+&prompt.root;<command>crm</command> configure primitive fake3 ocf:pacemaker:Dummy
+&prompt.root;<command>crm</command> configure primitive fake4 ocf:pacemaker:Dummy</screen>
+   </step>
+   <step>
+    <para>Check the status of the <command>crm status</command> command.
+    You should see something like the following:
+    </para>
+    <screen>TBD</screen>
+   </step>
+   <step>
+    <para>To move one of the Dummy primitives to run it on another node, use
+    the following command:</para>
+    <screen>TBD</screen>
+    <para>The status will change to this:</para>
+    <screen>TBD</screen>
+   </step>
+  </procedure>
+
+  <para>
+   To test whether fencing works, kill the <systemitem
+    class="daemon">pacemaker_remoted</systemitem> daemon and see how &pace;
```

Comment:
To test whether fencing works, kill the pacemaker_remoted daemon, and check how Pacemaker reacts: